js/product-details.js

```diff
@@ -1,161 +1,282 @@
+  }
+
+  const showAllElements = function (value, valuecontainerColumn, valuecontainerRow) {
+    value.forEach(function (element) {
+      showArrayElement(element, valuecontainerColumn)
+      showArrayElement(element, valuecontainerRow)
+    })
+  }
+
+  const showStringValue = function (value, containerColumn, containerRow) {
+    showValueInPosition(value, containerColumn, "column");
+    showValueInPosition(value, containerRow, "row");
+  }
+
+  const showValueInPosition = function (value, container, position) {
+    let valuecontainerRow = document.createElement("div");
+    valuecontainerRow.innerHTML = value;
+    container.appendChild(addPlacementClassesToElement(valuecontainerRow, position));
+  }
+
+  const showKey = function (key, container) {
+    let specialistrow = document.createElement("div");
+    specialistrow.innerHTML = key;
+    specialistrow.classList.add("details-notes--italic");
+    container.appendChild(specialistrow);
+  }
+
+  const showArrayElement = function (value, container) {
+    let textrow = document.createElement("div");
+    textrow.innerHTML = value;
+    textrow.classList.add("details-notes--margin");
+    container.appendChild(textrow);
   }
-  const productValuerow = document.getElementsByClassName(
-    "product-details-value"
-  )[0].children;
-  for (i = 0; i < productValuerow.length; i++) {
-    productValuerow[i].classList.add("product-details-value--row--none");
+
+  const addPlacementClassesToElement = function (elem, placing) {
+    elem.classList.add("product-details-value--" + placing);
+    elem.classList.add("product-details-value--" + placing + "--none");
+    elem.classList.add("product-details-value--" + placing + "--" + key);
+    return elem;
   }
-};
 
-const windowWidth = function () {
-  return document.getElementsByClassName("content-width")[0].offsetWidth;
-};
\ No newline at end of file
+  const addClickEventListenerToButtons = function () {
+    let productDetails = document.getElementsByClassName("details-button");
+    productDetails = [].map.call(productDetails, function (item) {
+      item.addEventListener("click", function () {
+        const SIZE_FOR_PRODUCT_DETAILS_IN_COLUMN = 769;
+        if (windowWidth() > SIZE_FOR_PRODUCT_DETAILS_IN_COLUMN) {
+          removeClass();
+          this.classList.add("details-button-active--row");
+          let valuecontainer = document
+            .getElementsByClassName(
+              "product-details-value--row--" + this.innerHTML
+            )[0]
+            .classList.toggle("product-details-value--row--none");
+        } else {
+          this.classList.toggle("details-button-active--column");
+          document
+            .getElementsByClassName(
+              "product-details-value--column--" + this.innerHTML
+            )[0]
+            .classList.toggle("product-details-value--column--none");
+        }
+      });
+    });
+  };
+
+  const removeClass = function () {
```

Comment:
if function named as removeClass it should take 2 parameters - element and class that we want to remove
in that case better to call it  toggleActiveTab or something like that, depends what exactly you mark as active